pkg/controller/install/install_controller.go

```diff
@@ -6,6 +6,7 @@ import (
 	"os"
 	"strings"
 
+	"github.com/go-logr/logr"
```

Comment:
I'd prefer to only use one type of logger. Let's stick with the `logf` generated by operator-sdk

Reply:
I am not introducing a new logger.  It is the one you have in `	log = logf.Log.WithName("controller_install")`.   It is just that when I pass it as parameter in a func, I need to import it.